subiquitycore/ui/form.py

```diff
@@ -304,7 +304,11 @@ class FormField(abc.ABC):
 
     def __init__(self, caption=None, help=None):
         self.caption = caption
-        self.help = help
+        # Allows styling at instantiation
+        if help is None or isinstance(help, tuple) or help is NO_HELP:
+            self.help = help
+        else:
+            self.help = ('info_minor',  help)
```

Comment:
My guess is that every time we pass in a string, we want to apply the 'info_minor', and every other case we can just assign to self.help.  What do you think about:

```suggestion
        if isinstance(help, str):
            self.help = ('info_minor',  help)
        else:
            self.help = help
```

Reply:
Yeah, I don't see any other use case. Let's go with the suggestion!